test/integration/user_visits_place_test.rb

```diff
@@ -0,0 +1,43 @@
+require "test_helper"
+class UserVisitsPlaceSpec < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test "with no previous entry" do
+    VCR.use_cassette("new place") do
+      visit "/"
+
+      page.fill_in 'nav-search', 
+        :with => 'Turing School of Software & Design'
+
+      click_button "Go"
+
+      assert_equal '/places/turing-school-of-software-design', 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.